[Other]: make init      scripts/../scripts/lib/color.sh: line 23: conditional binary operator expected

### What would you like to share?

git clone https://github.com/openimsdk/open-im-server && cd open-im-server 
make init

When I deployment component

===========> Initializing openim server project
/Users/*****/open-im-server/scripts/../scripts/lib/color.sh: line 23: conditional binary operator expected
make[1]: *** [gen.init] Error 2
make: *** [init] Error 2

### Additional information

**OS：** macOS Monterey 12.6.1
**Shell：** /bin/zsh (version 5.8.1)
**go version：** go1.21.1 darwin/amd64

--- a/scripts/lib/color.sh
+++ b/scripts/lib/color.sh
@@ -20,7 +20,7 @@
 ################################################################################
 
 # shellcheck disable=SC2034
-if [[ ! -v COLOR_OPEN ]]; then
+if [ -z "${COLOR_OPEN+x}" ]; then
     COLOR_OPEN=1
 fi
 

